src/main/java/controllers/DashboardController.java

```diff
@@ -0,0 +1,25 @@
+package controllers;
+
+import api.ApiClient;
+import api.request.GetDashboard;
+import helpers.ResponseHandler;
+import io.restassured.response.ValidatableResponse;
+import models.Dashboards;
+
+public class DashboardController {
+    ApiClient client;
+
+    public DashboardController(ApiClient client) {
+        this.client = client;
+    }
+
+    public ValidatableResponse getDashboards() {
+        return client.execute(new GetDashboard());
+    }
+
+    public String getDashboardName() {
+        Dashboards dashboards = ResponseHandler.getResponseAsClass(Dashboards.class,
+                getDashboards().extract().response());
+        return dashboards.getContent().get(0).getName();
```

Comment:
Why get(0)? Why not get(1)?
Maybe better get dasboards and send them to tests? Or create method
```boolean hasDashboardWithName(String dashboardName)```?

Reply:
refactor methods

Reply:
+ added hamcrest library and remade checking in the test